h/accounts/views.py

```diff
@@ -467,12 +467,10 @@ def disable_user(self):
         if err is not None:
             return err
 
-        username = appstruct['username']
-        pwd = appstruct['pwd']
+        user = User.get_by_userid(
```

Comment:
Don't we need to handle a case where the user is `None` / there is no authenticated user?